### PR TITLE
Allow both cumulative time limit and time limit per attempt

### PR DESCRIPTION
**Pull request by Nevseros**
* Allows both limits to be applied by the organization team.
* Adds an example to the Guidelines.

### CODE REVIEW

**Review by JoaoViniciusSantos — APPROVED** [review]

**Comment by Nevseros** [comment]
Should've asked before creating this PR, but, @thewca/software-team is it something that can easily be implemented on the WCA website/WCA Live if it's going to be merged?

**Comment by xsrvmy** [comment]
> and the cumulative time limit must not be greater than the time limit per attempt multiplied by the number of attempts.

This wording doesn't work if there is a cumulative limit across two or more events

**Comment by SAuroux** [comment]
> Should've ask before creating this PR, but, @thewca/software-team is it something that can easily be implemented on the WCA website/WCA Live if it's going to be merged?

Very likely not.

Question: what is the idea behind this? What sense does this make?

**Comment by Nevseros** [comment]
> Question: what is the idea behind this? What sense does this make?

See [this](https://forum.worldcubeassociation.org/t/proposal-allow-for-both-cumulative-time-limits-and-time-limits/16323) WCA Forum discussion.

**Comment by lgarron** [comment]
> > Should've ask before creating this PR, but, @thewca/software-team is it something that can easily be implemented on the WCA website/WCA Live if it's going to be merged?
> 
> Very likely not.

Thanks for your concern. I've talked to @gregorbg and it sounds like the WST should not be a blocker for adopting this change. That said, we would definitely want to make sure we are prepared by January to make this as easy as possible for Delegates using WCA software, and I'd be happy to put in some of my own time if it's needed.

File: wca-guidelines.md [hunk 1]
```diff
@@ -151,9 +151,11 @@ To be more informative, each Guideline is classified using one of the following
 
 ## <article-A><speedsolving><speedsolving> Article A: Speed Solving
 
+- A1a+) [CLARIFICATION] If both limits are applied, the time limit per attempt must not be greater than the cumulative time limit, and the cumulative time limit must not be greater than the time limit per attempt multiplied by the number of attempts.
```
Comment (Rouxles) [inline wca-guidelines.md]:
```suggestion
- A1a+) [CLARIFICATION] If both limits are applied, the time limit per attempt must not be greater than the cumulative time limit.
```

File: wca-guidelines.md [hunk 2]
```diff
@@ -151,9 +151,11 @@ To be more informative, each Guideline is classified using one of the following
...
 - A1a2+++) [ADDITION] If the result from an attempt is unavailable (e.g. the timer was reset), the judge should consult the WCA Delegate and make an estimate of the elapsed time for the attempt. This estimate counts towards the time limit, but must not be used as the basis of an official result for the individual attempt.
+- A1a2++++) [EXAMPLE] If a round has a cumulative time limit of 20 minutes, time limit for the attempt of 8 minutes and the competitor's results of the first two attempts are 7:00 and 7:30, then the time limit for the third attempt is the lowest time of 20:00 - 7:00 - 7:30 = 5:30 and 8:00, i.e. 5:30.
```
Comment (Rouxles) [inline wca-guidelines.md]:
```suggestion
- A1a2++++) [EXAMPLE] If a round has a cumulative time limit of 20 minutes and a time limit of 8 minutes and the competitor's results of the first two attempts are 7:00 and 7:30, then the time limit for the third attempt is the lowest time of 20:00 - 7:00 - 7:30 = 5:30 and 8:00, i.e. 5:30.
```

File: wca-regulations.md [hunk 3]
```diff
@@ -360,9 +360,9 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
...
+        - A1a2) For events other than 3x3x3 Fewest Moves and 3x3x3 Multi-Blind, the organization team may define a cumulative time limit. The cumulative time limit must either apply to all attempts in one round (e.g. 3 attempts with a cumulative time limit of 20 minutes), or to all attempts in a combination of rounds for different events. In this case, the time limit for a particular attempt is the lowest time limit of the announced time limit for the attempt and the cumulative time limit minus the total time in other attempts counting towards the time limit so far (see [Regulation A1a5](regulations:regulation:A1a5)). If a round has an applicable cumulative time limit, it must have at most one.
```
Comment (Rouxles) [inline wca-regulations.md]:
```suggestion
        - A1a2) For events other than 3x3x3 Fewest Moves and 3x3x3 Multi-Blind, the organization team may define a cumulative time limit. The cumulative time limit must either apply to all attempts in one round (e.g. 3 attempts with a cumulative time limit of 20 minutes), or to all attempts in a combination of rounds for different events. In this case, the time limit for a particular attempt is the the time limit or the cumulative time limit minus the total time in other attempts counting towards the time limit so far (see [Regulation A1a5](regulations:regulation:A1a5)), whichever is lower. If a round has an applicable cumulative time limit, it must have at most one.
```

File: wca-guidelines.md [hunk 4]
```diff
@@ -151,9 +151,11 @@ To be more informative, each Guideline is classified using one of the following
...
 - A1a2+++) [ADDITION] If the result from an attempt is unavailable (e.g. the timer was reset), the judge should consult the WCA Delegate and make an estimate of the elapsed time for the attempt. This estimate counts towards the time limit, but must not be used as the basis of an official result for the individual attempt.
+- A1a2++++) [EXAMPLE] If a round has a cumulative time limit of 20 minutes and a time limit of 8 minutes and the competitor's results of the first two attempts are 7:00 and 7:30, then the time limit for the third attempt is the lowest time of 20:00 - 7:00 - 7:30 = 5:30 and 8:00, i.e. 5:30.
```
Comment (Rouxles) [inline wca-guidelines.md]:
```suggestion
- A1a2++++) [EXAMPLE] If a round has a cumulative time limit of 20 minutes and a time limit of 8 minutes, and the competitor's results of the first two attempts are 7:00 and 7:30, then the time limit for the third attempt is the lowest time of 20:00 - 7:00 - 7:30 = 5:30 and 8:00, i.e. 5:30.
```

**Comment by dunkOnIT** [comment]
@gregorbg / @lgarron just checking if we need to work on the software implementation for this? Sorry if it's already being discussed and I missed it!

**Pull request merged**